RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -235,6 +235,16 @@ struct PurchaseInformation {
          return dateFormatter
      }()
 }
+
+extension PurchaseInformation: Identifiable {
+
+     var id: String {
+         return "\(productIdentifier)_\(Self.defaultDateFormatter.string(from: customerInfoRequestedDate))"
```

Comment:
I guess this could be enough, but I'm not totally convinced. I think it would be better to make it conform to `Hashable`, and then hash more properties into the `Hasher`. This way, you basically get conformance to `Equatable` and `Identifiable` (using `typealias ID = Self`) for free.

In addition, I'm thinking, what about a customer who purchased two products from different stores (app + web) that have the same `productIdentifier`? I'm not 100% that this situation could happen, but anyway we can easily cover for these edge cases.